src/osd/PeeringState.cc

```diff
@@ -3083,6 +3103,35 @@ void PeeringState::proc_replica_log(
   peer_missing[from].claim(std::move(omissing));
 }
 
+/**
+* Update min_last_complete_ondisk to the minimum
+* last_complete_ondisk version informed by each peer.
+*/
+void PeeringState::calc_min_last_complete_ondisk() {
+  ceph_assert(!acting_recovery_backfill.empty());
+  eversion_t min = last_complete_ondisk;
+  for (const auto& pg_shard : acting_recovery_backfill) {
+    if (pg_shard == get_primary()) {
+      continue;
+    }
+    if (peer_last_complete_ondisk.count(pg_shard) == 0) {
+      psdout(20) << " no complete info on: "
```

Comment:
All of these log lines should have the PeeringState::<method_name> at the start of the log line.

Reply:
Added "PeeringState::" to dout_prefix